ci.nix

```diff
@@ -13,6 +13,8 @@ pkgs.nix-community-infra // rec {
   build01-system = build01.system;
   build02 = importNixOS ./build02/configuration.nix "x86_64-linux";
   build02-system = build02.system;
-  build03 = importNixOS ./build02/configuration.nix "x86_64-linux";
+  build03 = importNixOS ./build03/configuration.nix "x86_64-linux";
   build03-system = build03.system;
+  build04 = importNixOS ./build04/configuration.nix "aarch64-linux";
+  build04-system = build04.system;
 }
```

Comment:
Actually it'd be ok for this file to be `default.nix`. 

> At the top-level of your repository, the agent will search for a Nix file to evaluate in the following order: nix/ci.nix, ci.nix or default.nix

 -- https://docs.hercules-ci.com/hercules-ci-agent/evaluation/
 
 It also documents the traversal of attributes.
 
 Next release series, 0.9.x, is updating this to include flake.nix support among other things.

Reply:
That said, renaming this to `ci.nix` is of course fine if CI is the whole purpose of this file.

Reply:
Yes, making the intent of this file clear, was my main motivation.